code/src/main/webapp/JavaScript/script.js

```diff
@@ -115,29 +153,38 @@ function startupGoogleLogin() {
   startApp;
 }
   
-var startApp = function() {
-  gapi.load('auth2', function() {
+var startApp = negativeUser => {
+  gapi.load('auth2', () => {
     // Retrieve the singleton for the GoogleAuth library and set up the client.
     auth2 = gapi.auth2.init({
       client_id: '1080865471187-u1vse3ccv9te949244t9rngma01r226m.apps.googleusercontent.com',
-      cookiepolicy: 'single_host_origin',
-      // Request scopes in addition to 'profile' and 'email'
-      //scope: 'additional_scope'
     });
-    attachSignin(document.getElementById('login-button-left-or-top'));
+    attachSignin(document.getElementById('login-button-left-or-top'), false);
+    attachSignin(document.getElementById('negative-login-button'), true);
   });
 };
 
-function attachSignin(element) {
-    console.log(element.id);
-    auth2.attachClickHandler(element, {},
-        function(googleUser) {
-          document.getElementById('name').innerText = "Signed in: " +
-              googleUser.getBasicProfile().getName();
-        }, function(error) {
-          alert(JSON.stringify(error, undefined, 2));
-        });
-  }
+function attachSignin(element, negativeUser) {
+  console.log(element.id);
+  auth2.attachClickHandler(element, {}, googleUser => {
+
+    document.getElementById('name').innerText = "Signed in: " + googleUser.getBasicProfile().getName();
+
+    const idtoken = googleUser.getAuthResponse().id_token;
```

Comment:
nit: idToken

Reply:
And I can tell why you're doing this, because the field on the param is idtoken, still worth keeping a consistent variable naming style with camel case.

Reply:
Changed both to idToken